src/home/live/index.jsx

```diff
@@ -3,7 +3,7 @@ import { Grid } from '@material-ui/core'
 
 import { NavigationButton } from 'home/live/navigation-button'
 
-export const Live = () => <Grid container justify={'center'} alignItems={'center'}>
+export const Live = () => <Grid item container justify={'center'} alignItems={'center'}>
```

Comment:
A material grid cannot be an item and a container at the same time. 

![image](https://user-images.githubusercontent.com/23569938/128852072-b769346c-19a7-44bf-89e2-ddcf3f61a9b9.png)
What I was referring to was this: Wrapping our grid containers within other grid items.

Let's do this throughout all the files touched in this PR. Maybe that will fix the css issue we discussed.

Reply:
Now it's done!